### ISSUE TITLE
Bulk Edit - Show existing user permissions

### ISSUE THREAD

**Issue by farhills** [sct/overseerr#2670]
### Description

I recently used the bulk edit feature to give my users permission to report issues.  I assumed that the permissions would follow 'standard' behaviour where existing permissions are inherited, and in a rush just ticked the boxes for report issues.  In the process, I nuked everyone's request permissions, including both the defaults as well as a few elevated users.

I eventually figured it out and got things sorted - not a huge deal, the feature just didn't work the way I expected.  

### Desired Behavior

It would be preferred if the Bulk Edit feature followed more 'standard' behaviour (eg. when changing windows file permissions) where existing permissions are inherited and shown on the page.  In the case of 'mixed' permissions (where only a subset of the selected users have been granted), these would be shown with a half-filled check box.  

In other words, only permissions explicitly changed by the admin should be modified.

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by sct** [comment]
We already show the users' permissions when you bulk edit, but ONLY if the users you are bulk editing already have the exact same permissions.

**Comment by farhills** [comment]
I guess from your perspective the permissions on the bulk edit form get written directly to the list of users?

Assuming that's the case then I would reword my request to skip writing to any fields where mixed permissions are detected.

**Comment by sct** [comment]
Not a bad idea. Would need to think about how we would implement that. I will leave this open to discussion to see what the rest of the team thinks then.

**Comment by danshilm** [comment]
I was planning on working on this sometime because it really limits the bulk edit functionality imo, and unfortunately without that it's not really as useful as it could be. And the half checkbox is exactly what I had in mind.
The bulk edit functionality also overwrites all existing user permissions, which I think it shouldn't do. Instead, it should add/remove the specified permissions from every user's permissions instead of overwriting them altogether.
